lib/templates/createRequestObject.js

```diff
@@ -52,6 +52,14 @@ const createRequestObject = ({ event }) => {
   req.rawHeaders = [];
   req.headers = {};
 
+  // Expose Netlify Function event and callback on request object.
+  // This makes it possible to access the clientContext, for example.
+  // See: https://github.com/netlify/next-on-netlify/issues/20
+  // It also allows users to change the behavior of waiting for empty event
+  // loop.
+  // See: https://github.com/netlify/next-on-netlify/issues/66#issuecomment-719988804
+  req.netlifyFunction = { event, context };
```

Comment:
i think this name is fine. maybe netlifyFunctionData since the value itself is not really a function but 🤷‍♀️

Reply:
True! How about `netlifyFunctionParams`, since it's not just data but also some functions?

Reply:
sounds good!!!!!

Reply:
It's now `netlifyFunctionParams` :)